documentation/code-gen/configure-typescript-sdk.md

```diff
@@ -0,0 +1,224 @@
+# Readme Configuration Guide for Typescript SDK
+This file describe how to configure readme files to make it available for Typescript SDK code generation.
+
+## Common Configuration
+Configure basic package information.
+
+### Basic Information
+Configure package title/description/tag.
+~~~~
+// file: readme.md
+
+``` yaml
+title: xxxxConfigurationClient
+description: xxxx Configuration Client
+openapi-type: arm
+tag: package-xxxx-xx-xx
```

Comment:
Should we add some name rule for tag to make it consistent? and give some guide for when we will use this tag

Reply:
I am fine to add some tag name convention for the consistency consideration. But should we discuss what kind of convention is needed in a more general way? For example. what the tag name would be in the scenarios of multi-package, in the scenarios of preview/stable version, in the scenarios of multi-api  etc ? Also I noticed that we use YYYY-mm to distinguish different versions previously, what if the service team has two api-versions such as 2020-05-01 and 2020-05-16 in one month?

Reply:
You can refer to changlong's PR for python

Reply:
PR #10123

Reply:
Okay. I will change it

Reply:
@akning-ms I have update the tag description like #10123 . Could you please review again? Thanks